app/packages/looker-3d/src/fo3d/utils.ts

```diff
@@ -125,7 +125,7 @@ export const getMediaPathForFo3dSample = (
 
 export const getFo3dRoot = (fo3dPath: string) => {
```

Comment:
Nice!! This function is missing TLC without a unit tests I think 😄 

A good example of App unit tests can be found [here](https://github.com/voxel51/fiftyone/blob/develop/app/packages/looker/src/overlays/classifications.test.ts). Tests can be colocated, e.g. `app/packages/looker-3d/src/fo3d/utils.test.ts`

App unit tests can be run in `./app` with `yarn test`

Reply:
This one has unit tests, but could use more coverage:

https://github.com/voxel51/fiftyone/blob/develop/app/packages/looker-3d/src/fo3d/tests/get-fo3d-root.test.ts

Reply:
I can add a couple tests with backslashes as well